generators/client/templates/vue/src/main/webapp/app/shared/date/filters.ts.ejs

```diff
@@ -1,13 +1,15 @@
 import Vue from 'vue'
-import moment from 'moment'
+import { format } from 'date-fns'
 
-export const DATE_FORMAT = 'YYYY-MM-DD';
-export const DATE_TIME_FORMAT = 'YYYY-MM-DD hh:mm';
+export const DATE_FORMAT = 'yyyy-MM-dd'
+export const DATE_TIME_FORMAT = 'yyyy-MM-dd HH:mm'
+
+export const DATE_TIME_LONG_FORMAT = "yyyy-MM-dd'T'HH:mm"
+export const INSTANT_FORMAT = "yyyy-MM-dd'T'HH:mm:ssZ"
+export const ZONED_DATE_TIME_FORMAT = "yyyy-MM-dd'T'HH:mm:ssXXXXX"
 
 export function initFilters() {
-  Vue.filter('formatDate', function(value) {
-    if (value) {
-      return moment(String(value)).format(DATE_TIME_FORMAT);
-    }
-  });
+    Vue.filter('formatDate', function(value){
+        return format(value, DATE_TIME_FORMAT);
```

Comment:
In case we agree on supporting only evergreen browser we could use `intl` date format instead of date-fns here. It would even format the date with the correct locale specific format

Reply:
I think its safe to assume that we only support evergreen browsers

Reply:
:heavy_check_mark: Will provide a separate PR for that.